src/applications/vaos/tests/containers/TypeOfCarePage.unit.spec.jsx

```diff
@@ -86,12 +89,34 @@ describe('VAOS <TypeOfCarePage>', () => {
     global.fetch.resetHistory();
   });
 
+  it('should call onClickedUpdateAddressButton after update button', () => {
```

Comment:
You can remove this test, I think everything is covered by the other test you added.

Reply:
Done! Thanks for showing how to write test for redux. Still a little fuzzy on writing redux testing but I'll search out for tutorials.